src/components/header/Header.js

```diff
@@ -0,0 +1,38 @@
+import { render } from '../../shared/dom';
+
+const template= (close, open) =>`
+    <header>
+      <img src="/static/img/logo.png" alt="logo" class="logo">
+      <div class="navBar">
+        <ul id="menu" class="menu">
+          <li>Movie</li>
```

Comment:
Add TODO that in the future we need to connect these elements with redirect function